specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```
@@ -0,0 +1,5723 @@
+          "$ref": "#/definitions/BlobInventoryCreationTime",
+          "description": "This property is used to filter objects based on the object creation time"
+        }
+      },
+      "description": "An object that defines the blob inventory rule filter conditions. For 'Blob' definition.objectType all filter properties are applicable, 'blobTypes' is required and others are optional. For 'Container' definition.objectType only prefixMatch is applicable and is optional."
+    },
+    "BlobInventoryCreationTime": {
+      "type": "object",
+      "properties": {
+        "lastNDays": {
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 36500,
+          "format": "int32",
+          "description": "When set the policy filters the objects that are created in the last N days. Where N is an integer value between 1 to 36500."
+        }
+      },
+      "description": "This property defines the creation time based filtering condition. Blob Inventory schema parameter 'Creation-Time' is mandatory with this filter."
+    },
+    "StorageAccountMigration": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Migration Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "description": "current value is 'default' for customer intiated migration"
+        },
+        "type": {
+          "type": "string",
+          "description": "SrpAccountMigrationType in ARM contract which is 'accountMigrations'"
+        },
+        "properties": {
+          "properties": {
+            "targetSkuName": {
+              "$ref": "./common.json#/definitions/SkuName",
+              "description":  "Target sku name for the account"
+            },
+            "migrationStatus": {
+              "type": "string",
+              "description": "Current status of migration",
+              "enum": [
+                "Invalid",
+                "SubmittedForConversion",
+                "InProgress",
+                "Complete",
+                "Failed"
+              ],
+              "x-ms-enum": {
+                "name": "migrationStatus",
+                "modelAsString": true
+              },
+              "readOnly": true
+            },
+            "migrationFailedReason": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Error code for migration failure"
+            },
+            "migrationFailedDetailedReason": {
+              "readOnly": true,
+              "type": "string",
+              "description": "Reason for migration failure"
+            }
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-client-name": "StorageAccountMigrationDetails",
+          "description": "The properties of a storage account’s ongoing or enqueued migration."
+        }
+      },
+      "required": [
+        "properties",
+        "targetSkuName"
+      ],
```

"targetSkuName" is a child property of "properties", so it cannot be added to the same level of required properties 
```suggestion
          "required": [
            "targetSkuName"
          ]
        }
      },
      "required": [
        "properties"
      ],
```